.github/workflows/wfc_pr_management.yml

```diff
@@ -28,21 +28,22 @@ jobs:
   open:
     if: github.event.pull_request && (inputs.matchLabel == '' || contains(github.event.pull_request.labels.*.name, inputs.matchLabel))
     runs-on: ubuntu-latest
-    env:
```

Comment:
isn't comment missing a token if you remove the default one?

Reply:
No: https://github.com/marocchino/sticky-pull-request-comment#github_token